dashboard/src/modules/components/OverviewComponent/NewRunsComponent.jsx

```diff
@@ -98,22 +96,28 @@ const NewRunsComponent = () => {
       onClick: () => dispatch(getMetaDataActions(dataset, "save", true)),
     },
     {
-      title: dataset.metadata[DASHBOARD_SEEN] ? "Mark unread" : "Mark read",
+      title: dataset.metadata?.global?.dashboard.seen
```

Comment:
What if `seen` hasn't been set? Should this be `dataset.metadata?.global?.dashboard?.seen`??

Reply:
Got it!